src/AWS.Deploy.DockerEngine/DockerFile.cs

```diff
@@ -79,10 +81,40 @@ public void WriteDockerFile(string projectDirectory, List<string>? projectList)
                 .Replace("{project-name}", _projectName)
                 .Replace("{assembly-name}", _assemblyName);
 
+            // Microsoft exposes 8081 along with 8080 in their .NET8 templates. I am preserving that behavior here when port 8080 is used.
+            if (_port == 8080)
```

Comment:
Since we made port configurable and they might choose port 5000 or something we should be honoring that here when we generate the dockerfile.